__tests__/redux/actions/socialLogin.test.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable prefer-promise-reject-errors */
+// /* eslint-disable prefer-promise-reject-errors */
+import configureMockStore from 'redux-mock-store';
+import moxios from 'moxios';
+import thunk from 'redux-thunk';
+import { socialLogin, loginWithFacebook, loginWithGoogle, loginWithTwitter } from '../../../src/redux/actions/socialLogin';
+
+require('@babel/polyfill');
+
+const mockStore = configureMockStore([thunk]);
+const Store = mockStore();
+
+describe('Action socialLogin', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+    Store.clearActions();
+  });
+  it('Should dispatch the action SIGNUP_SUCCESS', () => {
+    const accessToken = '45ytopmpowe';
+    const provider = {
+      username: 'claude',
+    };
+
+    const expectedResults = {
+      status: 200,
+      results: {
+        data: {
+          user: {
+            username: 'claude',
+          },
+        },
+      },
+    };
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: expectedResults,
+      });
+    });
+    Store.dispatch(socialLogin(accessToken, provider));
+    const action = Store.getActions();
+    expect(action[0]).toEqual(expectedResults[0]);
+    return Store.dispatch(socialLogin(accessToken, provider)).then(() => {
+    });
+  });
+  it('should let user login with twitter', () => {
```

Comment:
You shouldn't have this file since you are not using any action for social login